deploy/kustomize/base/kustomization.yaml

```diff
@@ -4,4 +4,3 @@ kind: Kustomization
 resources:
   - operator_group.yaml
   - subscription.yaml
-  - hco_cr.yaml
```

Comment:
the HCO CR is still needed to complete the deployment, also if it has an empty spec.

Reply:
fixed